### PR TITLE
Cps loader

### PR DESCRIPTION
**Pull request by greenjeffreym**
Inherit from HTMLElement instead of HTMLSpanElement

### CODE REVIEW

File: src/custom-elements/cps-loader/cps-loader.js [hunk 1]
```diff
@@ -5,17 +5,17 @@ import styles from './cps-loader.styles.css';
...
 class CpsLoader extends Component {
-	render(){
-		return (
-			<span className={`${styles.loader} ${this.props.customElement.page ? styles.page : ''}`}>
-				{
-					makeDots(this.props.customElement)
-				}
-			</span>
-		)
+    render(){
```
Comment (joeldenning) [inline src/custom-elements/cps-loader/cps-loader.js]: Looks like you changed from tabs -> spaces, but we (maybe unfortunately?) are a tabs-only company for frontend dev.

Comment (greenjeffreym) [inline src/custom-elements/cps-loader/cps-loader.js]: I switched my settings from spaces -> tabs, I think before I had extra spaces though.

Comment (joeldenning) [inline src/custom-elements/cps-loader/cps-loader.js]: Could you change this file to use spaces and then push that up, before merging this?

Comment (greenjeffreym) [inline src/custom-elements/cps-loader/cps-loader.js]: Yes, apparently VS Code needed one more setting to get the spaces to turn into tabs. Should be good now.

**Pull request merged**